src/client/Item/Item.ts

```diff
@@ -66,11 +57,15 @@ export class Item {
    * ({body: item} = await item.read<TodoItem>());
    * ```
    */
-  public read<T extends ItemDefinition>(options?: RequestOptions): Promise<ItemResponse<T>>;
-  public async read<T extends ItemDefinition>(options?: RequestOptions): Promise<ItemResponse<T>> {
+  public async read<T extends ItemDefinition = any>(options: RequestOptions = {}): Promise<ItemResponse<T>> {
     options = options || {};
-    if ((!options || !options.partitionKey) && this.partitionKey !== undefined) {
-      options.partitionKey = this.partitionKey;
+    if (!options.partitionKey) {
```

Comment:
Did we want to prevent folks from passing partition key in request options now?

Reply:
This applies in a few places, but I'm not gonna repeat my comment for them.

Reply:
I'm for it. Will do.

Reply:
I'm gonna make these annon objects now that they differ from `RequestOptions`

Reply:
Discussed on teams and going to leave this in two places for now. Will remove in a later PR https://github.com/Azure/azure-cosmos-js/issues/320